packages/service-library/src/servicelib/aiohttp/monitoring.py

```diff
@@ -107,18 +107,18 @@
 # http_request_latency_seconds_created{app_name="simcore_service_webserver",endpoint="/metrics",method="GET"} 1.641806371709292e+09
 
 
-kREQUEST_COUNT = f"{__name__}.request_count"
-kINFLIGHTREQUESTS = f"{__name__}.in_flight_requests"
-kRESPONSELATENCY = f"{__name__}.in_response_latency"
+kREQUEST_COUNT = f"{__name__}.request_count"  # noqa: N816
```

Comment:
maybe it requires a type to be `Final[str]` as well? I guess N816 is the lowercase "k"

Reply:
yes that relates to mixing upper/lower case